### PR TITLE
Update change log for 10.1.0

### PR DESCRIPTION
**Pull request by Chaoba**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+
+## Features ✨ and improvements 🏁
+* Set thread priorities and set thread CPU affinity based on the thread's priority ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+
+## Bug fixes 🐞
+* Fix rendering artifact when some of the model layer models may have wrong placement when globe view projection is used ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix rare heatmap flickering when zooming the map ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix an issue where an Observable event could be dispatched on a thread, different from the subscription thread ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix an issue where promoteId parameter for VectorSource was overwritten when source tilejson is loaded ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (ank27) [inline CHANGELOG.md:15]: fixes related to `promoteId ` and `glyph drawing`, `CPU affinity` were already mentioned in 10.1.0-beta.1 and rc.1, are these fixed again by gl-native or core?

Comment (kiryldz) [reply CHANGELOG.md]: @ank27 no they are not fixed again but taking into account that we're GA now we share all the updates (from beta, rc and stable) in this changelog. That will make easier for customers to see what changed between 10.0.0 and 10.1.0

@Chaoba taking it into account - can you append entries related to our SDK only from beta and rc release as well in this changelog?

Comment (Chaoba) [reply CHANGELOG.md]: No, just checked they are the same pr, but was included in the chagnelog of gl-native.  Will remove this line.

Comment (ank27) [reply CHANGELOG.md:17]: @kiryldz thanks for clearing that up 😄 .

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,32 @@
...
+* Fix android glyph drawing issue when 'high contrast' text accessibility feature is turned on ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix unexpectedly rapid map panning at high pitch level. ([#775](https://github.com/mapbox/mapbox-maps-android/pull/775))
+* Fix map move using faster map offsetting after zoom-in or zoom-out gesture. ([#738](https://github.com/mapbox/mapbox-maps-android/pull/738))
+* Fix annotation flickering and disappearing during dragging. ([#732](https://github.com/mapbox/mapbox-maps-android/pull/732))
+* Fix logo and attribution margin update. ([#744](https://github.com/mapbox/mapbox-maps-android/pull/744))
+* Fix `NullPointerException` while querying annotations. ([#746](https://github.com/mapbox/mapbox-maps-android/pull/746))
+* Limit fast fling gesture in a downwards direction when map is highly pitched. ([#754](https://github.com/mapbox/mapbox-maps-android/pull/754))
+* Fixed an issue that caused annotations not being updated in some cases. ([#753](https://github.com/mapbox/mapbox-maps-android/pull/753))
```
Comment (kiryldz) [inline CHANGELOG.md:25]:
```suggestion
* Fix an issue that caused annotations not being updated in some cases. ([#753](https://github.com/mapbox/mapbox-maps-android/pull/753))
```
@Chaoba can we bring more color what are 'some' cases?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,32 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.1.0 November 3, 2021
+
+## Features ✨ and improvements 🏁
+* Set thread priorities and set thread CPU affinity based on the thread's priority ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Introduce option to enable Multisample anti-aliasing (MSAA) for map rendering. ([#741](https://github.com/mapbox/mapbox-maps-android/pull/741))
```
Comment (kiryldz) [inline CHANGELOG.md:10]:
```suggestion
* Introduce option to enable multisample anti-aliasing (MSAA) for map rendering. ([#741](https://github.com/mapbox/mapbox-maps-android/pull/741))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,32 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.1.0 November 3, 2021
+
+## Features ✨ and improvements 🏁
+* Set thread priorities and set thread CPU affinity based on the thread's priority ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:9]: that's a bug

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,32 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.1.0 November 3, 2021
+
+## Features ✨ and improvements 🏁
+* Set thread priorities and set thread CPU affinity based on the thread's priority to improve overall map performance.([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Introduce option to enable multisample anti-aliasing (MSAA) for map rendering. ([#741](https://github.com/mapbox/mapbox-maps-android/pull/741))
+* Add convenience methods for stop expression. ([#698](https://github.com/mapbox/mapbox-maps-android/pull/698), [#764](https://github.com/mapbox/mapbox-maps-android/pull/764))
```
Comment (kiryldz) [inline CHANGELOG.md:10]:
```suggestion
* Add convenience methods for `stop` expression in Style DSL. ([#698](https://github.com/mapbox/mapbox-maps-android/pull/698), [#764](https://github.com/mapbox/mapbox-maps-android/pull/764))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,32 @@
...
+
+## Bug fixes 🐞
+* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:13]:
```suggestion
* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,32 @@
...
+* Fix rendering artifact when some of the model layer models may have wrong placement when globe view projection is used ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:14]:
```suggestion
* Fix rendering artifact when some of the model layer models may have wrong placement when globe view projection is used. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,32 @@
...
+* Fix rare heatmap flickering when zooming the map ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:15]:
```suggestion
* Fix rare heatmap flickering when zooming the map. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,32 @@
...
+* Fix an issue where an Observable event could be dispatched on a thread, different from the subscription thread ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:16]:
```suggestion
* Fix an issue where an Observable event could be dispatched on a thread, different from the subscription thread. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

File: CHANGELOG.md (hunk 1 above)
Comment (kiryldz) [reply CHANGELOG.md:17]:
```suggestion
* Fix an issue where promoteId parameter for VectorSource was overwritten when source tilejson is loaded. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,32 @@
...
+* Fix an issue where promoteId parameter for VectorSource was overwritten when source tilejson is loaded ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix android glyph drawing issue when 'high contrast' text accessibility feature is turned on ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```
Comment (kiryldz) [inline CHANGELOG.md:18]:
```suggestion
* Fix android glyph drawing issue when 'high contrast' text accessibility feature is turned on. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

**Review by ank27 — APPROVED** [review]

File: CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,32 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.1.0 November 3, 2021
```
Comment (Chaoba) [inline CHANGELOG.md:5]:
```suggestion
# 10.1.0 November 4, 2021
```

**Review by kiryldz — APPROVED** [review]

**Pull request merged**